crates/utils/re_video/src/demux/mp4.rs

```diff
@@ -186,6 +189,36 @@ fn codec_details_from_stds(
                 });
         }
     }
+    if let re_mp4::StsdBoxContent::Hev1(hvc1_box) = &stsd.contents {
```

Comment:
now that there's more than one case this should be part of a match statement joined with above condition and below default caase